content/rest-api/index.textile

```diff
@@ -1216,6 +1219,146 @@ Stats records contain a hierarchy of elements relating to messages, connections
 
 "See a complete example of a statistics response":/general/statistics.
 
+h2(#batch). Batch
+
+"Batch mode":/rest/batch enables you to submit multiple API requests for certain operations in a single API call. The API processes all the requests in parallel.
+
+h3(#batch-publish). Batch publish
+
+h6. POST rest.ably.io/messages
+
+Using this endpoint, you can publish messages to channels in parallel, by passing either a single @BatchSpec@ object or an array of @BatchSpec@ objects in the request body.
+
+A single @BatchSpec@ is an object of the form:
+
+bc[text]. {
+  channels: <channel names>,
+  messages: <messages>
+}
+
+* @<channel names>@ is a single channel name, or an array of up to 100 channel names, expressed as strings.
+* @<messages>@ is a single message, or an array of messages.
+
+The maximum size of each request body must be less than 2MiB and the total size of all messages in a @messages@ array must be less than the "message size limit":https://faqs.ably.com/what-is-the-maximum-message-size (64kiB by default for paid accounts, 16kiB for free accounts).
```

Comment:
We could update the message size limit link to: `/general/limits#message-size` and avoid linking out of the docs.

Reply:
Good idea. Addressed in [9c10c4e](https://github.com/ably/docs/pull/1231/commits/9c10c4e9f414030eb2a6423a3f99ad95f5b557ac).